meshroom/core/node.py

```diff
@@ -940,10 +940,15 @@ def endSequence(self):
         pass
 
     def stopComputation(self):
-        """ Stop the computation of this node. """
-        for chunk in self._chunks.values():
-            if not chunk.isExtern():
+        """ Stop the computation of this node. Chunks that are running locally will be stopped, and those that
+        are submitted locally will be reset.
+        """
+        for chunk in [c for c in self._chunks.values() if c.status.status == Status.RUNNING or
+                                                          c.status.status == Status.SUBMITTED]:
```

Comment:
this means a double loop. More simple to do the if inside the main loop.

Reply:
Done